packages/client/src/views/SysAdmin/Team/user/UserList.tsx

```diff
@@ -308,19 +318,36 @@ function UserListComponent(props: IProps) {
         setToggleActivation({
           ...toggleActivation,
           modalVisible: true,
-          selectedUser: user
+          selectedUser: null
         })
       } else {
         setToggleActivation({
           ...toggleActivation,
-          modalVisible: false,
-          selectedUser: null
+          modalVisible: false
```

Comment:
Why are we setting the selected user to null? This might break the behavior of the UserActivationModal.

Reply:
My mistake. Forgot to revert this change.